internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -130,3 +135,16 @@ func toMap(tags []*sdkcloudformation.Tag) map[string]string {
 	}
 	return m
 }
+
+// GetStackErrors returns the list of Cloudformation Resource Events, filtered by failures and erros.
+func (cf CloudFormation) GetStackErrors(conf StackConfiguration) ([]deploy.ResourceEvent, error) {
```

Comment:
nit: What do you think of renaming this to `ErrorEvents` as well? since it returns `[]deploy.ResourceEvent`.
This way the caller can also do `events, err := o.client.ErrorEvents(..)` instead of `errors, describeError`